.github/workflows/run_tests.yaml

```diff
@@ -12,11 +12,12 @@ on:
   push:
     branches:
       - master
+      - ac/nnunet_migration
 
   # Trigger current workflow on PR from ANY branch
   pull_request:
     branches:
-      - '*'
+      - '**'
```

Comment:
I had been wondering about this! (Thank you for finding the right syntax to get the run to trigger. I'll need to add this to SCT, too.)

Reply:
Through some digging I think what was said was that `*` won't work when the branch name has a `/` in it? But `**` will? I'll dig up the discussion again if I can

Reply:
That makes sense! And, we use `/` a _lot_ in SCT branch names, which would explain why PRs haven't been triggering against non-`master` branches.